rotkehlchen/tests/unit/accounting/test_staking.py

```diff
@@ -271,3 +277,118 @@ def test_validator_exit_pnl(
         ],
     )
     assert accountant.pots[0].pnls.taxable == ONE  # 1 ETH
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x0fdAe061cAE1Ad4Af83b27A96ba5496ca992139b']])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [FVal(3000)])  # ETH price at $3000
+@pytest.mark.parametrize('db_settings', [{
+    'eth_staking_taxable_after_withdrawal_enabled': True,
+}])
+def test_eth_withdrawal_processing(accountant: Accountant, ethereum_accounts: list[ChecksumEvmAddress]) -> None:  # noqa: E501
+    """Test ETH withdrawal processing with all different scenarios in realistic order:
+    1. Non-accumulating validator withdrawal (profit = amount)
+    2. Accumulating validator skimming withdrawal (profit = full amount)
+    3. Non-accumulating validator exit (profit = amount - MIN_EFFECTIVE_BALANCE)
+    4. Accumulating validator exit (profit = amount - MAX_EFFECTIVE_BALANCE)
+    """
```

Comment:
I believe you are missing a test for the case where you exit an accumulating validator with less than the MAX_EFFECTIVE_BALANCE